src/community/geopkg/pom.xml

```diff
@@ -42,16 +42,16 @@
       <artifactId>gs-wfs</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.geoserver.extension</groupId>
+      <artifactId>gs-geopkg-output</artifactId>
```

Comment:
This dependency is not reflected in the ``src/community/release/ext-geopkg.xml`` assembly file. Please update and make sure the community module package installs and runs correctly.

Reply:
I checked that, the ext-geopkg grabs this jar correctly because it uses a wildcard.